src/services/dex.js

```diff
@@ -24,4 +26,60 @@ export default {
       }
     });
   },
+
+  formOrderBook(asks, bids) {
+    const book = {
+      asks: [],
+      bids: [],
+    };
+
+    asks.forEach((l) => {
```

Comment:
`l`?

Reply:
HAHA! Ah dang it! I just copied this code over. I'll at it and come up with a more sensible name.

Reply:
Okay I fixed them.